.eslintrc.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
         "no-underscore-dangle": 0,
```

Comment:
Attention, set develop branch in place of master

Reply:
done